### PR TITLE
Remove bin files

### PR DESCRIPTION
**Pull request by ghe0Rghe**

### CODE REVIEW

File: src/utils.py [hunk 1]
```diff
@@ -32,10 +32,10 @@ def get_last_tag(cwd, _filter: str) -> Union[None, str]:
 
 def compose_build_commit_hash(cwd, tag, files: List, hash_limit=7) -> str:
     commits_hashes = []
-    cmd = ["git", "log", "--format=%h", "--", *files]
+    cmd = ["git", "log", "--format=%H", "--", *files]
```
Comment (slavict) [inline src/utils.py:35]: What improvements bring this change?

File: src/git_component.py [hunk 2]
```diff
@@ -138,6 +110,7 @@ def create_deb_package(gc, info, package_name, root_dir):
                 f.write(f"{k[4:]}: {v.strip()}\n")
         f.write(f"Version: {info.get('version')}\n")
         f.write(f"Package: {gc.name}\n")
+        f.write(f"Installed-Size: {package_size}\n")
```
Comment (slavict) [inline src/git_component.py:113]: Could you add also Download size for more deb package info?

File: src/git_component.py [hunk 3]
```diff
@@ -70,14 +12,16 @@
 import subprocess
 import sys
 from timeit import default_timer as timer
-from typing import List
+from typing import List, Dict
 
 import yaml
 
 from utils import slugify, write_cfg_file, compose_build_commit_hash, get_last_tag, gen_random_hash
```
Comment (slavict) [inline src/git_component.py]: please use pep8 as base for formating

File: src/git_component.py [hunk 4]
```diff
@@ -690,8 +623,161 @@ def run(self):
...
+        meta_data = self.file.get("package-info", {})
+        meta_data.update({
+            "version": package_version,
+            "name": self.name,
+            "build_ts": now_ts
+        })
+        write_cfg_file(os.path.join(src_dir, 'info.json'), meta_data, human_readable=True)
+        if arch_type:
+            create_package(self, meta_data, arch_type, package_dir, self.args.keep_storage_dir)
+        return 0
```
Comment (slavict) [inline src/git_component.py]: I supose that this return should be in run method not here

File: src/git_component.py [hunk 5]
```diff
@@ -690,8 +623,161 @@ def run(self):
...
+
+    def has_changes(self, files_to_check) -> bool:
```
Comment (slavict) [inline src/git_component.py]: annotation does not match with what really return method

File: src/utils.py (hunk 1 above)
Comment (ghe0Rghe) [reply src/utils.py:35]: -H is printing a full hash, and then the limit is taken from the CLI arguments. with -h it is limited to maximum 9.

**Pull request merged**